modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -103,16 +101,17 @@ class SampleCodeTest {
         let tempFolder = NSTemporaryDirectory().appending("cbllog")
         let config = LogFileConfiguration(directory: tempFolder) // <.>
         config.usePlainText = true // <.>
+        config.maxRotateCount = 2 // <.>
         config.maxSize = 1024 // <.>
         Database.log.file.config = config // <.>
-        Database.log.file.level = .info // <.>
+        Database.log.file.level = .verbose // <.>
         // end::file-logging[]
     }
     
     func dontTestNewFileLogging() throws {
         // tag::new-file-logging[]
         let tempFolder = NSTemporaryDirectory().appending("cbllog")
-        LogSinks.file = FileLogSink(level: .info, directory: tempFolder)
+        LogSinks.file = FileLogSink(level: .verbose, directory: tempFolder, usePlainText: false, maxKeptFiles: 2, maxFileSize: 1024)
```

Comment:
This is not a good example that will not generally work well and a lot of log will be missing.

Something like this will be better. The default max file size is not too bad.

```
LogSinks.file = FileLogSink(level: .verbose, directory: tempFolder, usePlainText: false, maxKeptFiles: 12)
```